Tasks/SonarQubePostTest/SonarQubeBuildBreaker.ps1

```diff
@@ -0,0 +1,176 @@
+. ./SonarQubeHelper.ps1
+
+$DefaultNumRetriesForAnalysisToComplete = 120; # there 1s delay between each retry plus the time it takes to make a web call
```

Comment:
Shouldn't this be configurable? How long is the server-side analysis expected to take for a large project (two minutes doesn't seem very long)?